capa/features/extractors/dnfile/helpers.py

```diff
@@ -87,9 +87,10 @@ def format_name(namespace: str, classname: str, methodname: str):  # type: ignor
 
 
 class DnProperty(DnClass):
-    def __init__(self, token: int, namespace: str, classname: str, propertyname: str):
+    def __init__(self, token: int, namespace: str, classname: str, propertyname: str, type: int):
         super(DnProperty, self).__init__(token, namespace, classname)
         self.propertyname: str = propertyname
+        self.type: int = type
```

Comment:
this should be an enum/class, we only track get/set (and undefined/other?) right now, but could easily extend it

Reply:
I've added the enum, but it is not an attribute of DnProperty anymore. I decided to remove it from DnProperty as it was only being set when the method (that accessed the property) belongs to the MethodDef table. For MemberRef and direct Field accesses, the property access type is being captured in a different way.